roles/hxr.monitor-galaxy/files/galaxy_monthly_active_users.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Description: This script is used to get the number of active users in Galaxy for the previous month.
+
+# Run the script on the 1st of every month (since Telegraf does not support something like that in the config, I am using a check here and the interval will be set to 24h)
+if [ "$(date +%d)" -ne 1 ]; then
+    exit 0
+fi
+
+# Get the previous month in YYYY-MM format
+prev_month=$(date -d "$(date +%Y-%m-01) -1 month" +%Y-%m)
+year=$(echo "$prev_month" | cut -d- -f1)
+month=$(echo "$prev_month" | cut -d- -f2)
+
+# Query gxadmin and format output for InfluxDB
+gxadmin csvquery monthly-users-active --year="$year" --month="$month" |  awk -F',' '{ printf "galaxy_monthly_active_users,month=%s active_users=%s\n", $1, $2 }'
```

Comment:
I think you can use iquery hereto get automatic telegraf output.

Reply:
I tested it but it didn't work and returned empty results so went with CSV query and parsed it.

Reply:
Mh, this looks like an gxadmin bug than. According to the doc it should work for most/all commands?

Reply:
It is not a bug, but the `iquery` magic is not created for this query. It's probably complex to develop, as the query has some customizations that will make the addition of that `iquery` magic a bit complex.